app/views/layouts/_header.html.erb

```diff
@@ -35,7 +35,7 @@
         <a class="nav-link navbar-item navbar-text <%= request.path == '/teachers' ? 'active' : '' %>" href="/teachers"><%= t("layout.link_to_teachers") %></a>
       </li>
       <li class="nav-item px-1">
-        <a class="nav-link navbar-item navbar-text <%= request.path == '/blog' ? 'active' : '' %>" href="https://blog.circuitverse.org/"><%= t("layout.link_to_blog") %></a>
+        <a class="nav-link navbar-item navbar-text <%= request.path == '/blog' ? 'active' : '' %>" href="https://blog.circuitverse.org/" target="_blank"><%= t("layout.link_to_blog") %></a>
```

Comment:
Why to redirect to new page?

Reply:
The blog webpage is opening on the same tab and not the new tab. Suppose we again want to go to the home page or any other webpage of the CircuitVerse website, now we can only move through the browser history arrows and not from the website itself as now the navbar is of blog website and not of CircuitVerse website. Since the blog webpage is inherited from the Blog repository and the page's header shows the content for blog webpage and not for the CircuitVerse website.

So, the Blog page is made to open on the new tab, as to differentiate the blog webpage from the CircuitVerse website.

**Blog**

Before

https://user-images.githubusercontent.com/84590758/153941783-937fc5ee-184a-4144-ade6-d7cff20379c1.mp4


After

https://user-images.githubusercontent.com/84590758/153941823-e147252e-1417-4230-9278-550bc4893de4.mp4

I mentioned these in the issue above

Thank You

Reply:
I think the page should redirect to new page. @gr455 WDYT?